scripts/get_access_token.py

```diff
@@ -0,0 +1,48 @@
+##!/usr/bin/env python
+
+import argparse
+import os
+from oauth2client.client import GoogleCredentials
+from oauth2client.service_account import ServiceAccountCredentials
+
+
+def get_access_token(json_credentials, new_scopes=None):
+    """I take a path to a service account's json credentials file and return an access token."""
+    scopes = ['https://www.googleapis.com/auth/userinfo.profile', 'https://www.googleapis.com/auth/userinfo.email']
+
+    if new_scopes is not None:
+        if isinstance(new_scopes, list):
+            scopes = scopes + new_scopes
+        else:
+            print("If you want to add additional scopes, add this in the form of a list.")
```

Comment:
This is a little unclear, maybe show the error or say the input is invalid because it's not a list, then continue with "If you want to add additional scopes .."

Reply:
good call, thanks!